electron/main.js

```diff
@@ -89,6 +89,15 @@ if (process.platform !== 'darwin') {
 }
 
 ///////////////////////////////////////////////////////////////////////////////
+// Fix indicator icon on Unity
+// Source: https://bugs.launchpad.net/ubuntu/+bug/1559249
+///////////////////////////////////////////////////////////////////////////////
+const de = process.env.XDG_CURRENT_DESKTOP;
+if (de.includes('Unity')) {
```

Comment:
Just skip using the variable `de` and instead `process.env.XDG_CURRENT_DESKTOP.includes..`